pages/blocks/in-article-cta/in-article-cta.js

```diff
@@ -0,0 +1,18 @@
+export default async function inArticleCta(block) {
+  const firstLayer = block.querySelector('div');
+
+  block.classList.add('content');
+  firstLayer.classList.add('in-article-cta-container');
+
+  if (firstLayer) {
+    const $divs = block.querySelector('div').querySelectorAll('div');
+
+    Array.from($divs).forEach(($div) => {
```

Comment:
Can be simplified to:
```suggestion
    block.querySelector('div').querySelectorAll('div').forEach(($div) => {
```